docs/content/misc/math-expr.md

```diff
@@ -195,3 +195,11 @@ See javadoc of java.lang.Math for detailed explanation for each function.
 | cartesian_fold(lambda,arr1,arr2,...) | folds a multi argument lambda across the cartesian product of all input arrays. The first arguments of the lambda is the array element and the last is the accumulator, returning a single accumulated value. |
 | any(lambda,arr) | returns 1 if any element in the array matches the lambda expression, else 0 |
 | all(lambda,arr) | returns 1 if all elements in the array matches the lambda expression, else 0 |
+
+
+## IP Address Functions
+| function | description |
```

Comment:
Since it's more convenient to type and it's reasonable that users will understand "ipv4" to mean an "IP address", I'm ok with using the shorter names you suggested.